loopy/check.py

```diff
@@ -773,7 +773,7 @@ def pre_schedule_checks(kernel, callables_table):
         check_multiple_tags_allowed(kernel)
         check_for_inactive_iname_access(kernel)
         check_for_write_races(kernel)
-        check_for_data_dependent_parallel_bounds(kernel)
+        # check_for_data_dependent_parallel_bounds(kernel)
```

Comment:
Why did you comment this out @tj-sun ?

Reply:
Okay, I think the problem is that we generate code where instructions depend on the same iname. 
Out-commenting this check does not actually cause any errors, so what I have done now is throwing a warning if there are data dependent parallel bounds and silence it on PyOP2 side.